presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -125,6 +137,7 @@ public String toString()
         StringBuilder builder = new StringBuilder();
         builder.append(schemaTableName).append(" ");
         Joiner.on(".").skipNulls().appendTo(builder, catalogName, schemaName, tableName);
+        builder.append(" columns=").append(columns);
```

Comment:
This will print like
```
columns=Optional.empty
```
or
```
columns=Optional[[orderkey:bigint:bigint],...]
```
It would look nicer to skip when not present
```java
columns.ifPresent(value -> builder.append(" columns=").append(value));